packages/gluestick/src/generator/templates/package.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+import type { CreateTemplate } from '../../types';
+
+module.exports = (createTemplate: CreateTemplate) => {
+  const template = createTemplate`
+   {
+      "name": "AppName",
+      "version": "1.0.0",
+      "description": "",
+      "main": "index.js",
+      "scripts": {
+        "start": "gluestick start",
+        "test": "gluestick test"
+      },
+      "dependencies": {
+        "gluestick": "../gluestick/packages/gluestick",
```

Comment:
Should not be hardcoded. Value of gluestick should be dependent on value from CLI - `--dev <path>`.